esp/scm/ws_workunits_struct.ecm

```diff
@@ -836,6 +836,7 @@ ESPstruct [nil_remove] WUResponseNote
     nonNegativeInteger ErrorCode;
     string Severity;
     nonNegativeInteger Cost;
+    [min_ver("1.95")] uint64 Id; // unique identifier for note
```

Comment:
@shamser should this be 2.01?

Reply:
Yes, it should be 2.01.  I've modified and squashed the changes.